pandas/tests/test_nanops.py

```diff
@@ -29,6 +29,220 @@ def skipna(request):
     return request.param
 
 
+@pytest.fixture
+def disable_bottleneck(monkeypatch):
+    with monkeypatch.context() as m:
+        m.setattr(nanops, "_USE_BOTTLENECK", False)
+        yield
+
+
+@pytest.fixture
+def arr_shape():
+    return 11, 7
+
+
+@pytest.fixture
+def arr_float(arr_shape):
+    np.random.seed(11235)
+    return np.random.randn(*arr_shape)
+
+
+@pytest.fixture
+def arr_float1(arr_float):
+    return arr_float
+
+
+@pytest.fixture
+def arr_complex(arr_float, arr_float1):
+    return arr_float + arr_float1 * 1j
+
+
+@pytest.fixture
+def arr_int(arr_shape):
+    np.random.seed(11235)
+    return np.random.randint(-10, 10, arr_shape)
+
+
+@pytest.fixture
+def arr_bool(arr_shape):
+    np.random.seed(11235)
+    return np.random.randint(0, 2, arr_shape) == 0
+
+
+@pytest.fixture
+def arr_str(arr_float):
+    return np.abs(arr_float).astype("S")
+
+
+@pytest.fixture
+def arr_utf(arr_float):
+    return np.abs(arr_float).astype("U")
+
+
+@pytest.fixture
+def arr_date(arr_shape):
+    np.random.seed(11235)
+    return np.random.randint(0, 20000, arr_shape).astype("M8[ns]")
+
+
+@pytest.fixture
+def arr_tdelta(arr_shape):
+    np.random.seed(11235)
+    return np.random.randint(0, 20000, arr_shape).astype("m8[ns]")
+
+
+@pytest.fixture
+def arr_nan(arr_shape):
+    return np.tile(np.nan, arr_shape)
+
+
+@pytest.fixture
+def arr_float_nan(arr_float, arr_nan):
+    return np.vstack([arr_float, arr_nan])
+
+
+@pytest.fixture
+def arr_float1_nan(arr_float1, arr_nan):
+    return np.vstack([arr_float1, arr_nan])
+
+
+@pytest.fixture
+def arr_nan_float1(arr_nan, arr_float1):
+    return np.vstack([arr_nan, arr_float1])
```

Comment:
Couldn't we just copy arr_nan as the second entry, e.g. 

```
@pytest.fixture
 def arr_nan_float1(arr_nan):
     return np.vstack([arr_nan, np.copy(arr_float)])
```

if you don't want to use the same object? Makes it a bit clearer. that the arrays have the same content.

Reply:
Good point. Since `arr_float1` was was redundant to `arr_float`, just removed `arr_float1`